Source/WTF/wtf/text/StringCommon.h

```diff
@@ -609,101 +607,50 @@ ALWAYS_INLINE const uint16_t* find16(const uint16_t* pointer, uint16_t character
     // If the pointer is unaligned to 16bit access, then SIMD implementation does not work. But ARM64 allows unaligned access.
     // Fallback to a simple implementation. We also use it for smaller memory where length is less than 16.
```

Comment:
This comment also looks stale, especially the "16bit access" part.  Maybe should be "character width"? 

Ditto for the "length is less than 16." part since your `thresholdLength` is 32.  Maybe should just use thresholdLength instead of 16 here?

Reply:
Changed. We no longer need this comment. Just mentioning to threshold.